src/__tests__/operate.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly refrain from using `eslint disable`, as that defies the purpose of using linters in the first place.